avocado/utils/archive.py

```diff
@@ -164,26 +164,30 @@ def extract(self, path='.'):
         """
         self._engine.extractall(path)
         if self.is_zip:
-            self._update_symlinks(path)
+            self._update_zip_extra_attrs(path)
 
-    def _update_symlinks(self, dst_dir='.'):
+    def _update_zip_extra_attrs(self, dst_dir):
         if platform.system() != "Linux":
```

Comment:
Actually these same attributes should be available on other UNIX platforms.  Since we're not testing this on platforms other than Linux, I actuaaly suggest to change the warning message bellow to something like "...only supported on Linux platforms".